booster/library/Booster/JsonRpc.hs

```diff
@@ -153,7 +154,17 @@ respond stateVar request =
 
                         solver <- maybe (SMT.noSolver) (SMT.initSolver def) mSMTOptions
                         result <-
-                            performRewrite doTracing def mLlvmLibrary solver substVars mbDepth cutPoints terminals substPat
+                            performRewrite
+                                doTracing
+                                def
+                                mLlvmLibrary
+                                solver
+                                substVars
+                                mbDepth
+                                cutPoints
+                                terminals
+                                rewriteOpts.interimSimplification
+                                substPat
```

Comment:
I think this is a bit too much in terms of parameters... 
Should we just use the `RewriteConfig` type (extended by `cutPoints`, `terminals`, `mbDepth` and `interimSimplificatoin`)?